torch/_dynamo/variables/torch.py

```diff
@@ -662,6 +662,26 @@ def fn_with_prim_types(x):
                 )
                 args = [stacked]
 
+            if self.value == torch.ops.inductor.accumulate_grad_.default:
+                arg0 = args[0]  # var
+                arg1 = args[1]  # grad
+                for grapharg in tx.output.graphargs:
+                    if grapharg.source == arg0.source:
+                        var_example = grapharg.example
+                for grapharg in tx.output.graphargs:
+                    if grapharg.source == arg1.source:
+                        grad_example = grapharg.example
```

Comment:
TODO(voz) - What about inlining? need an example.

Reply:
Self note - since this goes to zeros_like, you don't really need a real value. You can do something like this maybe? untested.

```
fgrad = arg1.as_proxy().node.meta['example_value']
var_example.grad = torch.zeros(fgrad.shape, device=fgrad.device)
```